ci/test/00_setup_env_win64.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win64 (bionic is used in the gitian build as well)
+export DOCKER_NAME_TAG=ubuntu:20.04
```

Comment:
the comment should stay to say that this is the version used in gitian as well. I presume this pull is blocked (waiting on) gitian getting bumped to focal? Thus the label "waiting for other".

Reply:
Oh, and this has to be bumped in the cirrus yaml as well (Sorry, confusing, but I didn't find a way around this redundancy)

Reply:
> Oh, and this has to be bumped in the cirrus yaml as well

Thanks. Did not realise that was required.

Reply:
Gitian has now been bumped.